src/adapter/bun/conninfo.test.ts

```diff
@@ -1,18 +1,25 @@
 import { Context } from '../../context'
 import { HonoRequest } from '../../request'
 import { getConnInfo } from './conninfo'
+import type { AddressType } from '../../helper/conninfo'
 
-const createRandomBunServer = () => {
-  const address = Math.random().toString()
-  const port = Math.floor(Math.random() * (65535 + 1))
+const createRandomBunServer = ({
+  address = Math.random().toString(),
+  port = Math.floor(Math.random() * (65535 + 1)),
+  family = 'IPv6',
+}: {
+  address?: string
+  port?: number
+  family?: AddressType | string
+} = {}) => {
```

Comment:
I was wondering if I should add a test case, but if it is not needed, I will remove it.